setup.py

```diff
@@ -43,6 +43,7 @@
         'vectormath>=0.2.0',
         'discretize>=0.4.0',
         'geoana>=0.0.4'
+        'scooby>=0.3.0',
```

Comment:
What do you think of making this a soft dependency as we did for matplotlib in discretize? Since it isn't strictly necessary for running simulations or inversions with SimPEG, I would be inclined to make it a soft rather than a hard dependency